### PR TITLE
Pixiv

### PR DESCRIPTION
**Pull request by ombe1229**

### CODE REVIEW

File: utils/pixiv.py [hunk 1]
```diff
@@ -166,10 +170,9 @@ async def cache_ranking_embed(self, mode):
...
         ]
-        done, _ = await asyncio.wait(embed_coroutine_list)
-        embed = [d.result() for d in done]
+        done = await asyncio.gather(*embed_coroutine_list)
 
-        await self.cache.set("pixiv_ranking_embed", embed)
+        await self.cache.set("pixiv_ranking_embed", done)
 
     async def latency(self):
         pixiv_latency1 = time.perf_counter()
```
Comment (kijk2869) [inline utils/pixiv.py:178]: Indent error

File: utils/pixiv.py [hunk 2]
```diff
@@ -116,7 +116,11 @@ async def make_illust_embed(info: PixivModel):
 
 async def make_ranking_illust_embed(info: PixivRankingModel):
     illust_url = await get_original_url(info.id)
```
Comment (kijk2869) [inline utils/pixiv.py:118]: Indent error

**Review by kijk2869 — CHANGES_REQUESTED** [review]
May be mobile bug.

**Review by kijk2869 — APPROVED** [review]

**Review by SaidBySolo — APPROVED** [review]

**Comment by SaidBySolo** [comment]
Merge into #76

**Pull request merged**